api/packages.go

```diff
@@ -71,6 +76,28 @@ func (p *Packages) Validate(repoOrViewName string, absDiretoryPath string) (*Val
 	return &report, nil
 }
 
+// ListInstalled returns a list of installed packages
+func (p *Packages) ListInstalled(repoOrViewName string) ([]InstalledPackage, error) {
+	var q struct {
+		Repository struct {
+			InstalledPackages []InstalledPackage
+		} `graphql:"repository(name: $repositoryName)"`
+	}
+
+	variables := map[string]interface{}{
+		"repositoryName": graphql.String(repoOrViewName),
+	}
+
+	graphqlErr := p.client.Query(&q, variables)
+
+	var installedPackages []InstalledPackage
+	if graphqlErr == nil {
+		installedPackages = q.Repository.InstalledPackages
+	}
+
+	return installedPackages, graphqlErr
```

Comment:
Generally acceptable to just do `return q.Repository.InstalledPackages, graphqlErr` here